web/static/js/components.js

```diff
@@ -2,142 +2,181 @@ import Vue from 'vue'
 
 const spinRate = 16 // milliseconds
 
-let nameForm = Vue.component('name-form', {
-  props: ['name'],
-  template: `
-    <form class="name">
-      <input type="text" v-model="name" maxlength="3" placeholder="___" />
-      <button type="submit" class="send"></button>
-    </form>
-  `
+let spinner = Vue.component('spinner', {
+	data: function() {
+		return {
+			interval: null
+		};
+	},
+	props: {
+		score: {
+			type: Number,
+			required: true,
+			default: 0
+		},
+		spm: {
+			type: Number,
+			default: 0
+		}
+	},
+	beforeCompile: function() {
+		this.interval = setInterval(() => {
+			this.score += this.increment
+		}, spinRate)
+	},
+	computed: {
+		prettyScore: function() {
+			return this.score.toFixed(3)
+		},
+		prettySPM: function() {
+			return this.spm.toFixed(1)
+		},
+		increment: function() {
+			return ((this.spm / 60) * (spinRate / 1000))
+		}
+	},
+	template: `<span class="score">{{ prettyScore }}</span>&gt;<span class="spm">{{ prettySPM }}</span>`
 });
 
-let scoreCounter = Vue.component('score-counter', {
-  data: function() {
-    return {
-      interval: null
-    };
-  },
-  props: {
-    score: { type: Number, required: true, default: 0 },
-    spm: { type: Number, default: 0 }
-  },
-  beforeCompile: function() {
-      this.interval = setInterval(() => {
-        this.score += this.increment
-      }, spinRate)
-  },
-  computed: {
-    prettyScore: function() {
-      return this.score.toFixed(3)
-    },
-    prettySPM: function() {
-      return this.spm.toFixed(1)
-    },
-    increment: function() {
-      return ((this.spm / 60) * (spinRate / 1000))
-    }
-  },
-  template: `<span>Score: {{ prettyScore }}, SPM: {{ prettySPM }}</span>`
-})
+let identity = Vue.component('identity', {
+	data: function(){
+		return {
+			name: '',
+			troupe: ''
+		}
+	},
+	props: ['name', 'troupe'],
+	computed: {
+		identity: {
+			get: function(){
```

Comment:
The default function is a getter anyway. There is no need to specify it as a getter. Please checkout the [basic example](https://vuejs.org/guide/computed.html).